test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -109,15 +88,6 @@ providers:
     files:
       - sourcePath: "./shared/v1alpha4/metadata.yaml"
     versions:
-      - name: v0.3.23 # latest published release in the v1alpha3 series; this is used for v1alpha3 --> v1beta1 clusterctl upgrades test only.
```

Comment:
Are we deprecating v1alpha3 upgrade tests? If yes, shall we also think of removing those from testgrid?

Reply:
Yes, after this is merged, we will need to remove them.